### PR TITLE
Add Slovak language

### PR DESCRIPTION
**Pull request by majko96**
SK

This PR/Translation is about #ISSUE_NUMBER.

**files translated**:
  - [x] `def/--LOCALE-CODE--/about.cson`
  - [x] `def/--LOCALE-CODE--/context.cson`
  - [x] `def/--LOCALE-CODE--/menu_darwin.cson`
  - [x] `def/--LOCALE-CODE--/menu_linux.cson`
  - [ ] `def/--LOCALE-CODE--/menu_win32.cson`
  - [ ] `def/--LOCALE-CODE--/settings.cson`
  - [x] `def/--LOCALE-CODE--/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale MY_LOCALE`

about.cson, context.cson, menu_darwin.cson, menu_linux.cson and welcome.cson are completely translated. 
to be continued... :)

### CODE REVIEW

File: def/sk/menu_linux.cson [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+      "C&opy":
+        value: "Kopírovať"
+      "Copy Pat&h":
+        value: "Kopírovať cestu"
+      "&Paste":
+        value: "Vložiť"
+      "Select &All":
+        value: "Vybrať všetko"
+      "Paste Without Reformatting":
+        value: "Vložiť bez preformátovania"
```
Comment (liuderchi) [inline def/sk/menu_linux.cson]: Please temporarily comment L55, L56 to pass CI; This menu item currently does not support translation

File: def/sk/menu_linux.cson [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        value: "Pridať zložku projektu…"
+      "Reopen Project":
+        value: "Znovu otvoriť projekt"
+        submenu:
+          "Clear Project History":
+            value: "Vymazať históriu projektu"
+      "Reopen Last &Item":
+        value: "Znovu otvoriť poslednú &položku"
+      "&Save":
+        value: "Uložiť"
```
Comment (liuderchi) [inline def/sk/menu_linux.cson]: For menu item containing `&` as hotkey hint, please also add `&` before the letter as your preferred hotkey.
(see no.3 in https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide)

e.g. https://github.com/liuderchi/atom-i18n/blob/83fdc78ee765f2a2cd879b70a0cd634429109d81/def/fr/menu_win32.cson#L5-L6

**Comment by majko96** [comment]
I think that now everything should be fine.... :)

**Comment by liuderchi** [comment]
@majko96 Slovak translation has released in latest atom-i18n `v0.21.0` 🎉
If you’d like to translate more into Slovak please refer issue #231

**Pull request merged**